### ISSUE TITLE
UI enhancement for disabling periodic jobs and improve state visibility 

### ISSUE THREAD

**Issue by dmclf** [hashicorp/nomad#19671]
### Proposal

Periodic jobs now have an `enabled` flag
Would it not be more convenient if this:
1) is a simple button on the UI, like `stop`, but then `disable` 
2) UI to be improved and clearly mark a job as disabled
  - now, when you disable a job it is still marked as 'running'
![image](https://github.com/hashicorp/nomad/assets/19852218/594bbe17-231d-421c-b0b2-46bb6c185560)
even though the enabled is false, which may not visually match expectations.
3) additionally, can this state also be exported to the prometheus metrics, so it is also easy to make monitoring/alerting on disabled jobs if people would desire/require?

### Use-cases
More simplified way to disable a job, with better visibility.
(also suits cases where some people that may not feel comfortable editing raw jobspecs, would still be able to disable a job)

**Comment by jrasell** [comment]
Hi @dmclf and thanks for raising this issue. I think it makes sense to have a way to show if a periodic job is disabled or not within the UI, therefore I have pass this over to our UI engineer who will be able to better understand the request for roadmapping.

**Comment by philrenaud** [comment]
@dmclf I gave this some thought and put together an experiment with it this afternoon, and the end result is I've opted away from it. I wanted to explain about why:
- The "stop" and "start" nomenclature matches how we use job commands on the CLI and the property of our http API
- The "Stop" / "Are you sure" / "Start" buttons would deviate from other non-periodic jobs if we changed them up here, requiring users to become familiar with another pattern.

As such, I don't think this is worth doing. I thank you for raising this issue and for the idea.
